test/test_geci_distance.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+import numpy as np
+from geci_distance import *
+
+
+class Test_geci_distance(unittest.TestCase):
+    def setUp(self):
+        """
+        Crea variables que se usarán en las pruebas
+        """
+        self.X: int = 10
+        self.Sigma = 2
+        self.Beta: int = 0
```

Comment:
Cambiando a:
```suggestion
        self.Beta: int = 1
```
y el `self.assertAlmostEqual(output, 0.18126924)` ya cazas dos mutantes más